server/webhook_parser.go

```diff
@@ -289,21 +294,35 @@ func appendCommentNotifications(wh *webhook, verb string) {
 		wh.notifications = append(wh.notifications, notification)
 	}
 
-	// Don't send a notification to the assignee if they don't exist, or if are also the author.
-	// Also, if the assignee was mentioned above, avoid sending a duplicate notification here.
+	commentMessage := fmt.Sprintf("%s **commented** on %s:\n>%s", commentAuthor, jwh.mdKeySummaryLink(), jwh.Comment.Body)
+
+	// Don't send a notification to the assignee or reporter if they don't exist, or if they are also the author.
+	// Also, if the assignee or reporter was mentioned above, avoid sending a duplicate notification here.
 	// Jira Server uses name field, Jira Cloud uses the AccountID field.
-	if assigneeMentioned || jwh.Issue.Fields.Assignee == nil || jwh.Issue.Fields.Assignee.Name == jwh.User.Name ||
-		(jwh.Issue.Fields.Assignee.AccountID != "" && jwh.Issue.Fields.Assignee.AccountID == jwh.Comment.UpdateAuthor.AccountID) {
-		return
+	if !assigneeMentioned && shouldSendNotification(jwh.Issue.Fields.Assignee, jwh.User.Name, jwh.Comment.UpdateAuthor.AccountID) {
+		wh.notifications = append(wh.notifications, webhookUserNotification{
+			jiraUsername:  jwh.Issue.Fields.Assignee.Name,
+			jiraAccountID: jwh.Issue.Fields.Assignee.AccountID,
+			message:       commentMessage,
+			postType:      PostTypeComment,
+			commentSelf:   jwh.Comment.Self,
+			recipientType: recipientTypeAssignee,
+		})
+	}
+	if !reporterMentioned && shouldSendNotification(jwh.Issue.Fields.Reporter, jwh.User.Name, jwh.Comment.UpdateAuthor.AccountID) {
+		wh.notifications = append(wh.notifications, webhookUserNotification{
+			jiraUsername:  jwh.Issue.Fields.Reporter.Name,
+			jiraAccountID: jwh.Issue.Fields.Reporter.AccountID,
+			message:       commentMessage,
+			postType:      PostTypeComment,
+			commentSelf:   jwh.Comment.Self,
+			recipientType: recipientTypeReporter,
+		})
```

Comment:
It looks like if the reporter is the assignee, they will receive two notifications. Can we avoid this somehow?